ggshield/core/scan/commit.py

```diff
@@ -12,6 +11,11 @@
 _RX_HEADER_LINE_SEPARATOR = re.compile("[\n\0]:", re.MULTILINE)
 
 
+REGEX_HEADER_INFO = re.compile(
```

Comment:
See :label: 1

Reply:
For this one and the next one, I would prefer exposing a function which would take a string to parse and return a Python object instead of a raw regex. That would be higher-level. Maybe in another PR.